docs/src/code-examples/setup-install/go.mdx

```diff
@@ -0,0 +1,7 @@
+---
+lang: go
+---
+
+```shell nonumbers
+go get -u github.com/govau/notify-client-go
```

Comment:
Once we tag the initial version we should update this I think. And/or remove the `-u` flag. Point being that installing with Go modules should be a little less cowboy.